packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -28,18 +39,48 @@ const selectTargetAccountPrompt = async (
         };
       });
   } else {
+    let sandboxUsage = {};
+    try {
+      const accountId = getAccountId(defaultAccountConfig.portalId);
+      sandboxUsage = await getSandboxUsageLimits(accountId);
+    } catch (err) {
+      logger.debug('Unable to fetch sandbox usage limits: ', err);
```

Comment:
Only added a debug statement here, we have another sandbox limit validation after this prompt runs with more descriptive messaging